### ISSUE TITLE
MakeInstall.bat doesn't show ANY devices

### ISSUE THREAD

**Issue by Otus9051** [corpnewt/gibMacOS#236]
I am trying to use Clover and use makeinstall.bat for install the OS image to my usb. Problem is nothing shows up. I have also tried manually using makeinstall.py and using administrator privileges (even though it asks for one every time). Nothing happened. I deleted all partitions from my usb, tried replugging it, tried NTFS and Fat32. Nothing shows up. Also I did try the Show All disks option which still didn't do anything,

**Comment by Otus9051** [comment]
Commit Number `4083ca20785d96ef26eecac89c1c9e37532beb44` seems more broken as it doesn't even open. FYI I am using Python 3.9

**Comment by Otus9051** [comment]
Never.

**Comment by brendoncdrew** [comment]
Damn

**Comment by cutie-lyta** [comment]
The reason it is broken, on my end at least, is the removal of wmic.exe on windows 11, and I think later version of 10
It might still work on 7 and 8.1, didn't test tho
I am trying to switch to the powershell `Get-WMIObject` command, but I don't know how to use powershell so I have no luck unfortunately

But at least I know where it's from, if we didn't already

**Comment by VioSynthax** [comment]
Seems that this is still unusable on Windows 11.

**Comment by Otus9051** [comment]
Fixed on mine tho

**Comment by VioSynthax** [comment]
> Fixed on mine tho

Weird. Any idea what fixed it?

**Comment by Otus9051** [comment]
nope. no idea. actually, try installing the media feature pack maybe that helped it?

**Comment by wilkergrcv** [comment]
If you are using an external hard drive instead of a flash drive, change this line in the MakeInstall.py file.

By default it will be like this:

def get_disks_of_type(self, disk_list, disk_type=(0,2)):

with this configuration makeinstall will only show removable and unknown devices

now just change that same line with a 3 looking like this

def get_disks_of_type(self, disk_list, disk_type=(0,2,3)):

save the file and run MakeInstall.bat again and be happy.

**Comment by RavHilton** [comment]
Same issue here - no drives are shown (Win 11, latest public build, latest public updates, Python 3.11.1, x64) and the addition of disk_type "3" as suggested by https://github.com/wilkergrcv makes no difference at all. Yes, already running elevated as admin...

Even after using option "D", all I see is as follows:

#######################################################

Potential Removable Media
#######################################################

!WARNING! This list includes ALL disk types.
!WARNING! Be ABSOLUTELY sure before selecting
!WARNING! a disk!

Q. Quit

Usage:
